awx/main/tests/data/inventory/plugins/satellite6/files/foreman.yml

```diff
@@ -3,19 +3,19 @@ compose:
   ansible_ssh_host: foreman['ip6'] | default(foreman['ip'], true)
 group_prefix: foo_group_prefix
 keyed_groups:
-- key: foreman['environment_name'] | lower | regex_replace(' ', '') | regex_replace('[^A-Za-z0-9\_]', '_') | regex_replace('none', '')
```

Comment:
I don't know about making changes to these files, because they are reference files which are machine-generated. If there's anyway we could ignore them instead, that would be preferable